### DISCUSSION TITLE
Set your email notifications

### DISCUSSION THREAD

**Discussion by AlexanderPico** [wikipathways/GPML-Repo#23] [General]
If you want email notifications for all discussion activity here (perhaps similar to how you had it setup for google groups), then do the following:

1. Click "Watch" on this repo (above), choose "Custom" and check "Discussions" along with any other preferences.
2. Go to your notification settings (https://github.com/settings/notifications) and check "Email" under the sectio called "Watching."

**Comment by ariutta** [comment]
Selecting "All Activity" also works, if you want to be notified for any activity on this repo. If you don't want all notifications, then the option Alex described allows you to choose which updates you get.
